app/views/games/_team_list.html.haml

```diff
@@ -1,15 +1,14 @@
 - if teams.nil? or teams.size.eql? 0
-  .zero-items-text
-    %h4= t('game.team_list.no_teams')
+  .zero-items-text No Teams
 - else
   - max = 0
   %table.table.table-bordered.table-striped
     %thead
       %tr
         %th{ :style => "width:50px;" } #
-        %th=t('game.team_list.team_name') # The team's name
-        %th{ :style => "width:100px;" }= t('game.team_list.team_achievements') # The team's achievements
-        %th{ :style => "width:100px;" }= t('game.team_list.team_points') # The number of points the team has
+        %th Name
+        %th{ :style => "width:100px;" } Achievements
+        %th{ :style => "width:100px;" } Points
```

Comment:
This looks to revert the I18n work that Camden did in his other PR. We should use the `I18n` version of the strings instead of these.